specification/communication/data-plane/Email/preview/2023-15-01-preview/CommunicationServicesEmail.json

```diff
@@ -0,0 +1,413 @@
+            "in": "header",
+            "name": "repeatability-request-id",
+            "description": "If specified, the client directs that the request is repeatable; that is, that the client can make the request multiple times with the same Repeatability-Request-Id and get back an appropriate response without the server executing the request multiple times. The value of the Repeatability-Request-Id is an opaque string representing a client-generated, globally unique for all time, identifier for the request. It is recommended to use version 4 (random) UUIDs.",
+            "type": "string",
+            "required": true
+          },
+          {
+            "in": "header",
+            "name": "repeatability-first-sent",
+            "description": "Must be sent by clients to specify that a request is repeatable. Repeatability-First-Sent is used to specify the date and time at which the request was first created in the IMF-fix date form of HTTP-date as defined in RFC7231. eg- Tue, 26 Mar 2019 16:06:51 GMT",
+            "type": "string",
+            "required": true
+          },
+          {
+            "in": "header",
+            "name": "x-ms-client-request-id",
+            "description": "Tracking ID sent with the request to help with debugging.",
+            "type": "string",
+            "required": false,
+            "format": "uuid",
+            "x-ms-client-name": "ClientRequestId"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "body",
+            "name": "message",
+            "description": "Message payload for sending an email",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/Message"
+            }
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "Error",
+            "headers": {
+              "x-ms-error-code": {
+                "description": "Error code - this will be the same as the code in the error property in the response body.",
+                "type": "string"
+              }
+            },
+            "schema": {
+              "$ref": "../../../Common/stable/2021-03-07/common.json#/definitions/CommunicationErrorResponse"
+            }
+          },
+          "202": {
+            "description": "Message accepted, the `Operation-Location` header contains the URL to poll for the message status.",
+            "headers": {
+              "Repeatability-Result": {
+                "description": "Status of a repeatable request.",
+                "type": "string"
+              },
+              "Operation-Location": {
+                "description": "Location url of where to poll the status of this message from.",
+                "type": "string"
+              },
+              "Retry-After": {
+                "description": "Amount of time client should wait before retrying the request, specified in seconds ",
```

Comment:
Why would a client retry a successful 202 request?

Reply:
take a look into this whether we need it.

Reply:
Removed this from swagger. We're not using this in the response for SendEmail.
Updated in the next iteration.